projects/meta_cl/experiments/dendrites.py

```diff
@@ -20,11 +20,23 @@
 
 from copy import deepcopy
 
-from networks import DendriticNetwork
+from networks import ANMLDendriticNetwork, DendriticNetwork
+from nupic.research.frameworks.vernon import MetaContinualLearningExperiment, mixins
 
+from .anml_replicate import ANMLTransform, metacl_anml_replicate
 from .oml_replicate import metacl_oml_replicate
 
 
+class DendritesExperiment(mixins.RezeroWeights,
```

Comment:
I am curious of the performance with and without rezeroweights (dense vs sparse). I saw in the comment below it is not really sparse now because it never calls the `post_batch` method, but at some point later we can evaluate.

Reply:
Good point. As of now, it doesn't get past 0 accuracy (both meta-training and testing), so we'll need a working network for comparison.

Reply:
Maybe that's what's making it harder to train now. I wouldn't doubt if it's too over parameterized without the sparse weights.